### ISSUE TITLE
Logo Uses incorrect font.

### ISSUE THREAD

**Issue by DylanBulmer** [DylanBulmer/TheDocs#7]
**I'm submitting a ...**
  - [x] bug report
  - [ ] feature request

**What is the current behavior?**

The logo does not grab the correct font.

**What is the expected behavior?**

The logo to use Google's Open-Sans as a font, I need to fix the `fonts.css` to be able to use every font file for it.

**Please tell us about your environment:**
  
  - Version: 0.1.3
  - OS: MacOS, Windows, Linux

**Comment by EricEisaman** [comment]
Could it be that you are missing the glyph for the Open Sans Font as a result of the way Inkscape performed the SVG export?
![missing_glyph](https://user-images.githubusercontent.com/6116865/38667933-da451ba0-3e10-11e8-8e59-b41ed5bdd4ee.png)

I noticed the Adobe Illustrator SVG export plugin for the Maven build system adds glyphs on export.  Perhaps Inkscape has a similar mode, setting, or plugin available.
[SVG Export Plugin for Adobe Illustrator](https://github.com/midonet/midonet-docs/blob/master/tools/midonet-docs-maven-plugin/src/main/resources/images/cloud/openstackextension-cover.st)

**Comment by DylanBulmer** [comment]
I have access to Illustrator so I can see if I can move the svg file over and use the plugin. I just worked with it yesterday to create my new logo for the app.

**Comment by DylanBulmer** [comment]
I think I fixed the issue in Illustrator, I will find out once I release the next version. I will close for now and reopen if I need to.

**Comment by DylanBulmer** [comment]
I realized that instead of using the physical font files for my logo, I could "expand" the text, making the letters into objects in illustrator. I then moved some anchor points in the newly expanded objects to line up the letters better and make the "C" and the "S" in DOCS connect instead of having this weird overlap. I also united some letters together to make a better flow so there are no seen gaps between letters at different sizes. Now it is done and works on every OS.
